libdevice/include/asan_libdevice.hpp

```diff
@@ -0,0 +1,164 @@
+//==-- device-sanitizer-report.hpp - Structure and declaration for assert
+// support --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+
+#include <cinttypes>
+
+#if OMP_LIBDEVICE && INTEL_COLLAB
```

Comment:
Is this right? Do we have any public documentation we can point to?

Reply:
Hi, @AllanZyne 
Please remove all INTEL_* macro and omp related code, we only need sycl part in inte/llvm repo sycl branch.

Reply:
done